src/components/pages/LandingPage.vue

```diff
@@ -7,6 +7,8 @@ import TheContainer from '@/components/common/TheContainer.vue';
 import WelcomeSection from '@/components/views/WelcomeSection.vue';
 import MovieCard from '@/components/views/MovieCard.vue';
 import MoviesSummary from '@/components/views/MoviesSummary.vue';
+import NoMatch from '@/components/views/NoMatch.vue';
```

Comment:
It doesn't seem to be used anywhere in this component's template so I think we can remove it 🤔

Reply:
Yeah, I used it in this template for the tests purposes and then forgot to remove it from script 🤦‍♀️